src/Altinn.Notifications.Email/appsettings.json

```diff
@@ -1,4 +1,11 @@
 {
+  "KafkaSettings": {
+    "BrokerAddress": "localhost:9092",
+    "EmailSendingConsumerSettings": {
+      "ConsumerGroupId": "altinn-notifications-email-sending",
```

Comment:
could the consumer group id match the component name altinn-notifications-email for simplicity? Or are you planning multiple consumer groups within this micro service?

Reply:
I'm assuming there will be at least one more consumer.